apis/grafana/v1alpha1/datasource_types.go

```diff
@@ -52,9 +70,26 @@ type DatasourceList struct {
 	Items           []Datasource `json:"items,omitempty" protobuf:"bytes,2,rep,name=items"`
 }
 
+type DatasourcePhase string
+
+const (
+	DatasourcePhaseProcessing  DatasourcePhase = "Processing"
+	DatasourcePhaseTerminating DatasourcePhase = "Terminating"
+	DatasourcePhaseSuccess     DatasourcePhase = "Success"
+	DatasourcePhaseFailed      DatasourcePhase = "Failed"
+)
+
```

Comment:
Use GrafanaPhase instead of DatasourcePhase/DashboardPhase
And also update in also dashboard_types.to

Reply:
Updated with `GrafanaPhase`